pyclient/examples/demo_run_script.py

```diff
@@ -0,0 +1,36 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+"""Demo how to run a Python script on the server and open a table generated by the script."""
+import pandas as pd
+
+from examples.import_test_data import import_taxi_records
+from pydeephaven import Session, Table
+
+
+def run_script(dh_session: Session) -> Table:
+    server_script = '''
+t2 = t.where("VendorID > 0")\
```

Comment:
If it doesn't affect the execution, would it make sense to indent the script a bit?  I think if we end the line in a way that it makes it evident for python that the script has not finished (like after an open parenthesis) we may be able to avoid the use of backslash and also indent the code in the script to make it a bit more readable.  I am thinking something like

```
t2 = t.where(
     "VendorID > 0"
).sort(
     "VendorID", "fare_amount"
).headBy(
     5, "VendorID"
)
```

I am not actually sure this is better or not.   Really just throwing it out there as an idea.  I used a style like that a bit in a doc I wrote to split long queries in more legible pieces.

Reply:
Yes, definitely